test/powershell/engine/ResourceValidation/TestRunner.ps1

```diff
@@ -0,0 +1,57 @@
+function Test-ResourceStrings
+{
+    param ( $AssemblyName, $ExcludeList )
+
+    # determine the needed resource directory. If these tests are moved
+    # this logic will need to change
+    $repoBase = (Resolve-Path (Join-Path $psScriptRoot ../../../..)).Path

```

Comment:
Is it better to use: git rev-parse --show-toplevel?

Reply:
sure - I'll make that change

Reply:
Cannot assume that we have git tools. For example, the Code Coverage machine does not have git tools.

Reply:
in that case, this is fine